docs/rest_api.md

```diff
@@ -608,6 +616,32 @@ Authorization requirements:
 
 * Administrative access.
 
+#### node_set_metadata
+
+`PUT /node/<node>/metadata/<label>`
+
+Request Body: 
+
+	{
+		"value": <value>
+	}	
```

Comment:
Hm, think it might be cleaner to just have the entire body of the request be the metadata. @henn, thoughts?